examples/misc/tickedstroke_demo.py

```diff
@@ -0,0 +1,83 @@
+"""
+============================
+TickedStroke Patheffect Demo
+============================
+
+"""
+
+import matplotlib.patches as patches
+from matplotlib.path import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patheffects as PathEffects
+
+# Direct to path
+plt.figure(figsize=(6, 6))
+ax1 = plt.subplot(111)
```

Comment:
The canonical way of doing this nowadays is
~~~
fig, ax1 = plt.subplots(figsize=(6, 6))
~~~

Reply:
Done.